### PR TITLE
Port Button, ExpandToggle, and Video components

### PR DESCRIPTION
**Pull request by alexgoff**

### CODE REVIEW

File: packages/epo-react-lib/src/atomic/Accordion/Accordion.stories.tsx [hunk 1]
```diff
@@ -4,6 +4,24 @@ import Accordion from ".";
 
 const meta: ComponentMeta<typeof Accordion> = {
   component: Accordion,
+  argTypes: {
```
Comment (alexgoff) [inline packages/epo-react-lib/src/atomic/Accordion/Accordion.stories.tsx:7]: went back and improved some Storybook content

File: packages/epo-react-lib/src/atomic/Button/Button.test.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import { render, screen, within } from "@testing-library/react";
+import Button from "./Button";
+
+const text = "Stuff and Things";
+const children = <span>{text}</span>;
+const className = "some-class";
+const id = "some-id";
+
+describe("Button", () => {
```
Comment (alexgoff) [inline packages/epo-react-lib/src/atomic/Button/Button.test.tsx:9]: copied the button tests from rubin-obs-client

File: packages/epo-react-lib/src/atomic/Button/Button.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    },
+    ref
+  ) => (
+    <Styled.Button
+      $isBlock={isBlock}
+      $styleAs={styleAs}
+      $hasIcon={!!icon}
+      aria-disabled={isInactive || undefined}
+      data-testid="button"
+      {...{ ...(buttonProps as any), className, ref }}
```
Comment (alexgoff) [inline packages/epo-react-lib/src/atomic/Button/Button.tsx:41]: the shape expected by `styled-components` does not map perfectly to the shape defined by `HTMLProps<HTMLButtonElement>` so the remaining props get cast as `any`

File: packages/epo-react-lib/src/styles/base/index.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import { css } from "styled-components";
+import { aButton, aButtonTheme } from "@/styles/mixins/appearance";
+
+export default css`
```
Comment (alexgoff) [inline packages/epo-react-lib/src/styles/base/index.ts:4]: grab a couple of the base styles from `rubin-obs-client` and add them to `GlobalStyles`

File: packages/epo-react-lib/src/atomic/Button/Button.tsx [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    >
+      <>
+        {icon && (
+          <IconComposer
+            icon={icon}
+            size={iconSize}
+            aria-hidden={children && true}
```
Comment (alexgoff) [inline packages/epo-react-lib/src/atomic/Button/Button.tsx:48]: this was previously set as `role='presentation'` if children were present. Changed this to `aria-hidden` to follow recommendations not to change the implicit role of SVG's and that `role='presentation'` [does not actually hide the element from screenreaders](role='presentation')

**Review by blnkt — APPROVED** [review]

**Pull request merged**